src/main/java/com/amazon/randomcutforest/serialize/RandomCutForestSerDe.java

```diff
@@ -77,6 +87,29 @@ public String toJson(RandomCutForest rcf) {
         return gson.toJson(rcf);
     }
 
+    /**
+     * Serializes a RCF object to ion binary.
+     *
+     * @param rcf a RCF object
+     * @return ion binary serialized from the RCF
+     * @throws IonException when the input cannot be serialized
+     */
+    public byte[] toIon(RandomCutForest rcf) {
+        try {
+            IonReaderBuilder readerBuilder = IonReaderBuilder.standard();
+            IonReader reader = readerBuilder.build(toJson(rcf));
+            IonObjectMapper mapper = new IonObjectMapper();
+            mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
```

Comment:
Why do we need to set this?

Reply:
Since during serialization random objects are removed and represented by empty objects, without which they won't be de-serialized and be left to null.